packages/validator/contracts/BaseExecutor.sol

```diff
@@ -11,8 +11,12 @@ import "@openzeppelin/contracts/contracts/utils/introspection/ERC165.sol";
  */
 abstract contract BaseExecutor is IExecutor, ERC165 {
     address public immutable validator;
+    address public issuer;
```

Comment:
I don't think these belong in `BaseExecutor`; I can certainly imagine executors that handle claims for multiple issuers. This should definitely be in a subclass.

Reply:
The `configData` can just be individual variables in the executor subclasses, too.

Reply:
Makes sense. Do you think it's better to define a separate subclass like`SingleIssuerExecutor` or just put it in the ERC20Executor?